azurerm/resource_arm_kubernetes_cluster.go

```diff
@@ -209,6 +209,48 @@ func resourceArmKubernetesCluster() *schema.Resource {
 				Set: resourceAzureRMKubernetesClusterServicePrincipalProfileHash,
 			},
 
+			"network_profile": {
```

Comment:
can [we document this block/these new fields here](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/kubernetes_cluster.html.markdown#L127)? (I've had to link to the Blame since you can't link to a regular line in a Markdown file in Github ¯\_(ツ)_/¯)